src/Definition.php

```diff
@@ -209,7 +207,7 @@ public function current()
     /**
      * Iterator: current item key
      *
-     * @return int|string
+     * @return null|int|string
```

Comment:
I don't think key can be null

Reply:
It can be.

```php
$definition = new \Laminas\Server\Definition();
self::assertNull($definition->key());
```

See also `Return Values` on https://www.php.net/key

Reply:
Oh yeah, haven't considered empty set possibility